README.md

```diff
@@ -98,6 +98,20 @@ You can run `npm test` to execute the test suite and linters. To help you develo
 ### The examples
 The examples are hosted within the docs folder and are ran in the simple add that loads the Datepicker. To extend the examples with a new example, you can simply duplicate one of the existing examples and change the unique properties of your example.
 
+## Accessibility
+
+### Keyboard support
+
+* *Left*: Move to the previous day.
+* *Right*: Move to the next day.
+* *Up*: Move to the previous week.
+* *Down*: Move to the next week.
+* *PgUp*: Move to the previous month.
+* *PgDn*: Move to the next month.
+* *Home*: Move to the previous year.
+* *End*: Move to the next year.
+* *Enter/Esc/Tab*: close the calendar. (Enter & Esc calls preventDefautl)
+
```

Comment:
@bcullman, @mattgoucher: It would be really nice if the ability to enter invalid dates was no longer needed; the ability to enter/retain invalid dates has gotten somewhat more complicated in the latest release; we're diverging a bit further from the upstream parent project...